Formula/skaa.rb

```diff
@@ -0,0 +1,41 @@
+class Skaa < Formula
+  desc "Seven Kingdoms: Ancient Adversaries"
+  homepage "https://7kfans.com"
+  url "https://github.com/the3dfxdude/7kaa/releases/download/v2.15.5/7kaa-2.15.5.tar.xz"
+  sha256 "350a2681985feb4b71d20677d1a6d11921b9e3d97facbc94e4f020e848d8ab2b"
+  license "GPL-2.0-only"
+
+  # ARM is unsupported
+  depends_on arch: :x86_64
+
+  depends_on "enet"
```

Comment:
```suggestion
  depends_on "enet"
```
Are you sure this isn't because of your hard-coded compiler path?

Reply:
Sure, with `Clang` we could also compile for M1 and x86, however the dev himself isn't sure the game runs without problems on M1. I don't have a M1 machine myself to verify either. As such, it's safer to just limit the game to x86 architectures.

When compiling with Clang, the multiplayer options are disabled, due to Clang (through the configure script) not properly identifying my processor as an x86 processor. This problem doesn't show up when using `gcc`. 

If the x86 processor is not properly identified, the configure script disables multiplayer.

Reply:
You only need the full path to `gcc` because you have not used `fails_with` correctly.